config/initializers/statsd.rb

```diff
@@ -1,13 +1,31 @@
 require 'statsd'
 
-config_file = Rails.root.join('config/statsd.yml')
-raise "No such file: config/statsd.yml" unless File.exist?(config_file)
+module StatsdLoader
+  def self.create
+    config = config_from_env || config_from_file
+    raise('Could not initialize statsd by file or by ENV') if config.nil?
```

Comment:
you could also do `config = config_from_env || config_from_file || raise( ... )` but probably shouldn't